src/PullRequestMetricsTable.tsx

```diff
@@ -165,12 +169,27 @@ const makeColumns = (gitHubBaseUri: string): GridColDef[] => [
     type: 'boolean',
     width: 125,
   },
-  { field: 'author', headerName: 'Author' },
+  {
+    field: 'author',
+    headerName: 'Author',
+    renderCell(params: GridRenderCellParams<string>) {
+      const { author } = params.row;
+      return (
+        <ReactDomLink to={`/${gitHubHostname}/users/${author}`} replace>
```

Comment:
Is there a way to have this work with the styling that mui's `Link` component provides?